pkg/virt-handler/isolation/detector.go

```diff
@@ -268,30 +257,25 @@ func getPPid(pid int) (int, error) {
 	return process.PPid(), nil
 }
 
-func (s *socketBasedIsolationDetector) getSlice(pid int) (controllers []string, slice string, err error) {
-	slices, err := s.cgroupParser.Parse(pid)
-	if err != nil {
-		return
-	}
+// consider reusing getMemoryOverhead()
+// This is not scientific, but neither what libvirtd does is. See details in:
+// https://www.redhat.com/archives/libvirt-users/2019-August/msg00051.html
+func getMemlockSize(vm *v1.VirtualMachineInstance) (int64, error) {
```

Comment:
Hm... this looks to me like a leftover after rebase. Or is this function used somewhere?

Reply:
You're right, thanks for catching that!